etc/config/fortran.amazon.properties

```diff
@@ -2,6 +2,7 @@ compilers=&gfortran_86
 defaultCompiler=gfortran81
 demangler=/opt/compiler-explorer/gcc-8.1.0/bin/c++filt
 objdumper=/opt/compiler-explorer/gcc-8.1.0/bin/objdump
+compilerType=fortran
```

Comment:
This is superflous, as it is defined on the default properties

Reply:
It didn't seem to work for me without this line?

Reply:
They exist in the amazon files to improve readability, so you are right, it's probably better to leave this here :)